scania-digital-design-system/tegel
# feat(dropdown): removed old dropdown

The old dropdown was removed and replaced with the one previously called dropdown-v2

BREAKING CHANGE: The old dropdown was removed and replaced with the one previously called dropdown-v2

**Solving issue**  
Fixes: #

**How to test**  
1. Go to storybook
2. Check in Components -> Dropdown
3. Test the controls and check the style of the dropdown.

**Suggested test steps**
